src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -699,7 +699,10 @@ private void RenderPackageFactoryViewModel_PropertyChanged(object sender, Proper
                     // A full regeneration is required to get the edge geometry.
                     foreach (var vm in Watch3DViewModels)
                     {
-                        vm.RegenerateAllPackages();
```

Comment:
BTW, I have used Dynamo 2.12.0 when I debug Dynamo code, I found that "Settings" had disappeared from the menu. I don't know how to toggle "ShowEdges" now.

Reply:
Thank you @ZiyunShang , we are doing a lot of UI improvements in Dynamo and indeed show edges are currently hidden. It will be back later but pending for implementation now.